src/commands/signal2020.js

```diff
@@ -157,9 +158,12 @@ class Signal2020Command extends TwilioClientCommand {
   }
 }
 
+const baseFlags = { ...TwilioClientCommand.flags };
```

Comment:
Can you add `Run twilio profiles:list for more info`

Reply:
Done